pkg/util/kube.go

```diff
@@ -377,6 +378,31 @@ func GetPodLabelValues(n, label string) (map[string]string, error) {
 	return m, nil
 }
 
+// GetPodAnnotations gets a map annotations from a pod name for the given: namespace, pod label and a timeout for checking the pod annotations
+func GetPodAnnotations(n, podName string, timeout int) (map[string]string, error) {
```

Comment:
What do you think about this?
```suggestion
func GetPodAnnotations(n, podName string, retries int) (map[string]string, error) {
```

Reply:
better